cpp/pixels-common/include/physical/FilePath.h

```diff
@@ -18,29 +18,25 @@
  * <https://www.gnu.org/licenses/>.
  */
 
-/*
- * @author gengdy
- * @create 2024-11-19
- */
+//
```

Comment:
Do not edit the author's information. Be careful when merging mini-pixels code into pixels.